docs/cel2/whats-changed/op-l2.md

```diff
@@ -26,28 +27,23 @@ More details on supported transaction types [here](https://specs.celo.org/tx_typ
 
 ## L1 fees
 
-In the Optimism model, an extra fee is added on to user transactions in order to cover the cost of the L1, this can be surprising to users since they have no visibility about what that fee may be since it is not included in the results of calling `eth_estimateGas`.
+In the Optimism model, an extra fee is added in order to cover the cost of transactions on the L1. This can be surprising to users as it is not included in the results of calling `eth_estimateGas` and is challenging to predict.
```

Comment:
```suggestion
In the Optimism model, an extra fee is added in order to cover the cost of transactions on the L1. This can be surprising to users, as it is not included in the results of calling `eth_estimateGas`, and is challenging to predict.
```
```suggestion
In the Optimism model, an extra fee is added in order to cover the cost of transactions on the L1. This can be surprising to users, as it is not included in the results of calling `eth_estimateGas`, and is challenging to predict.
```